components/companies-using-kedro/companies-using-kedro.module.scss

```diff
@@ -17,100 +17,74 @@
 
 .inner {
   background: $color-dark;
+  flex-direction: column;
   font-size: 3rem;
   font-weight: 700;
   line-height: 6rem;
   display: flex;
   margin: 0 auto;
   max-width: $max-content;
-
-  @include breakpoint(medium-only) {
-    flex-direction: column;
-  }
-
-  @include breakpoint(small) {
-    flex-direction: column;
-    font-size: 1.5rem;
-    line-height: 3rem;
-  }
 }
 
-.left {
-  align-items: center;
-  justify-content: right;
-  display: flex;
-  padding-right: 1.5rem;
-  width: 45%;
+.sectionTitle {
+  @include h1;
 
-  @include breakpoint(medium-only) {
-    justify-content: left;
-    width: 100%;
-  }
+  margin-bottom: 8rem;
+  text-align: center;
 
   @include breakpoint(small) {
-    justify-content: left;
-    width: 100%;
+    margin-bottom: 2.5rem;
+    padding: 0;
   }
 }
 
 .right {
```

Comment:
maybe we can remove this and incorporate styling in the other div.inner?

Reply:
hey good spot, I just pushed my changes to remove them now